stdlib/bool.mli

```diff
@@ -42,7 +42,7 @@ external ( || ) : bool -> bool -> bool = "%sequor"
 (** {1:preds Predicates and comparisons} *)
 
 val equal : bool -> bool -> bool
-(** [equal b0 b1] is [true] iff [b0] and [b1] are both either [true]
+(** [equal b0 b1] is [true] if and only if [b0] and [b1] are both either [true]
     or [false]. *)
```

Comment:
This specific one feels strange with the English expanded:
```suggestion
(** [equal b0 b1] is [true] only if [b0] and [b1] are both [true]
    or both [false]. *)
```

Reply:
I'm not sure what you find odd here with "if and only if" but not odd with "iff".  Don't you read "iff" aloud as "if and only if"?

In either case I agree the sentence is a bit awkward because it is mathematical style.  Something like the following would flow better: "[equal b0 b1] is [true] if the Booleans [b0] and [b1] are equal (that is, both are [true] or both are [false]), and [false] if [b0] and [b1] differ (that is, one is [true] and the other is [false])."  But it's a different style, more verbose, and not necessarily better than the mathematical style.

Reply:
> Would it be fine just to have "only if"?

Or just "if", which often means "if and only if" in informal speech.  In both cases: no, it's too imprecise.

Reply:
It's more that I don't read it aloud, so when reading the text I scan "iff" quickly, but the full words take longer to scan over, given the foibles of my own damaged eyesight. But that's not a good enough reason to lose the mathematical precision!